### ISSUE TITLE
Community Tech Tree Support

### ISSUE THREAD

**Issue by GonDragon** [LouisB3/HabTech-Classic#17]
A lot of people uses this tech tree, so it should support it. But as for now it wasn't updated for 1.3, it's a low priority task.

**Comment by LouisB3** [comment]
Yes, I use it personally and would like to support it. At the moment I'm not aware of any HabTech parts that are so much more advanced than stock that they need to be placed in CTT's nodes, but I'm open to suggestions.

**Comment by GonDragon** [comment]
I should take a closer look to CTT and how the parts of this mod are currently disposed in the stock tech tree. If I remember well, there are some early tech nodes in CTT that could contain some parts, but I'm just speculating. When CTT comes out for 1.3, I will see if some part fits in any of it's categories and then suggest a new disposal (or not).

**Comment by Nick2253** [comment]
I've added my take for CTT support in pull request #21.
